### PR TITLE
chore: build arm64 docker images

### PR DESCRIPTION
**Pull request by steveiliop56**
See #108

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -7,25 +7,35 @@ on:
...
       
-    - name: Login to GitHub Container Registry
-      uses: docker/login-action@v2
-      with:
-        registry: ghcr.io
-        username: ${{ github.repository_owner }}
-        password: ${{ secrets.GITHUB_TOKEN }}
-          
-    - name: Push image to GitHub
-      if: github.ref == 'refs/heads/master'
```
[the remark this answers is not in the document]
Comment (steveiliop56) [reply .github/workflows/docker-image.yml:28]: Oh I forgot to include that part of the code. Let me fix it.

[the remark this answers is not in the document]
Comment (steveiliop56) [reply .github/workflows/docker-image.yml:28]: Ah there is a small problem, you see the buildx action does everything in one go, so it builds and pushes. So I can't really tell it to push only on this condition.

File: .github/workflows/docker-image.yml [hunk 2]
```diff
@@ -7,25 +7,35 @@ on:
...
+          registry: ghcr.io 
+          username: ${{ github.repository_owner }} 
+          password: ${{ secrets.GITHUB_TOKEN }} 
+
+      - name: Build and push image 
+        uses: docker/build-push-action@v6 
+        with: 
+          context: . 
+          platforms: linux/amd64, linux/arm64 
+          push: true
```
Comment (mawise) [inline .github/workflows/docker-image.yml:38]: Could we add a condition here?

`push: github.ref == 'refs/heads/master'`

or similar?

Comment (steveiliop56) [reply .github/workflows/docker-image.yml:38]: Huh that may actually work

Comment (steveiliop56) [reply .github/workflows/docker-image.yml:38]: Let's try

**Comment by mawise** [comment]
Thank you @steveiliop56 !

**Pull request merged**